.changelog/24541.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:security
+security: add CT executeTemplate to default function_denylist
```

Comment:
"CT" might not be immediately obvious to non-HashiCorp folks reading the changelog:

```suggestion
security: add executeTemplate to default template function_denylist
```